images/dnsmasq/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM centos:7
```

Comment:
no UBI option?

Reply:
No, I checked but ubi doesn't have `dnsmasq` default :(

Reply:
so no base UBI that can have DNSmasq installed from the repos?

Reply:
@gbraad no whatever UBI provide as default repo `dnsmasq` is not present only `bind` is there.

Reply:
@praveenkumar can we inquire someone about this? So for now this is OK to use, but I would think it is better over time to use a RHEL base (especially for downstream).